docs/samples/browser-plugin-meetings/app.js

```diff
@@ -560,6 +563,13 @@ function doPostMediaSetup(meeting) {
 
 async function joinMeeting({withMedia, withDevice} = {withMedia: false, withDevice: false}) {
   const meeting = webex.meetings.getAllMeetings()[selectedMeetingId];
+
+  meeting.on('meeting:llm:connected', () => {
```

Comment:
i am not sure if we need to name it llm as developer does not know anything about it 

meeting:socket:connected

thoughts ?

Reply:
@arun3528 does LLM replace the existing mercury connection that gets created when registering a device?

Reply:
@sreenara - No. They are two different socket connections. Both exist in the browser. So, should we name it like `meeting:transcription:connected` ?

Reply:
@mkesavan13 so with this we'll have `meeting:transcription:connected` and `meeting:transcription:started` that a developer has to listen for?

Reply:
Connected is to know that the socket is connected. Transcription started will be received when they toggle the transcription

Reply:
Sounds good @mkesavan13 . Let's go ahead with this.